src/bots/depositor.py

```diff
@@ -262,3 +253,44 @@ def prepare_and_send_tx(self, module_id: int, quorum: list[DepositMessage]) -> b
         label = 'success' if success else 'failure'
         MODULE_TX_SEND.labels(label, module_id).inc()
         return success
+
+    def _fetch_actual_messages(self) -> list[BotMessage]:
+        # Fetch messages and apply filters
+        actualize_filter = self._get_message_actualize_filter()
+        prefix = self.w3.lido.deposit_security_module.get_attest_message_prefix()
+        sign_filter = get_messages_sign_filter(prefix)
+
+        return self.message_storage.get_messages_and_actualize(lambda x: sign_filter(x) and actualize_filter(x))
+
+    def _get_preferred_to_deposit_modules(self) -> List[int]:
+        # gather quorum
+        now = datetime.now()
+        for module_id in variables.DEPOSIT_MODULES_WHITELIST:
+            if self._get_quorum(module_id):
+                self._quorum_cache[module_id] = now
+
+        module_ids = self.w3.lido.staking_router.get_staking_module_ids()
+        modules = [
+            module for module in self.w3.lido.staking_router.get_staking_module_digests(module_ids) if self._is_module_depositable(module)
+        ]
+        sorted_modules = sorted(
+            modules,
+            key=lambda module: self._validator_difference(module),
+        )
+        return list(map(lambda x: x[2][0], sorted_modules))
```

Comment:
1. Sort modules by `_validator_difference`.
2. Form a list of modules until find healthy module.
e.g. response will be
[(CSM, healthy)]
or
[(CSM, unhealthy), (sDVT, healthy)]
or 
[(CSM, unhealthy), (sDVT, unhealthy), (curated, healthy)]